src/pages/address/fragments/transactions/AddressTransactions.scss

```diff
@@ -211,3 +211,10 @@
     }
   }
 }
+
+#no-transaction {
```

Comment:
by conversion, the id needs to be in camelCase
It's better to use class than iD because it's reusable